### ISSUE TITLE
MT4 support

### ISSUE THREAD

**Issue by bgabela** [wait4signal/sharing-is-caring#2]
your product works great so far.
Is it possible to provide a version to for MT4-->MT5 MT5-->MT4 

--_Submitted by Patrick Ortbauer on mql5.com_

**Comment by venomdev** [comment]
I don't know if this would be a priority for a free project. I believe that MetaQuotes want to stop supporting MT4 altogether in the future.

https://www.mql5.com/en/forum/324399

and looking at this comment: https://www.mql5.com/en/forum/324399#comment_26594412

MetaQuotes might not be offering new MT4 licenses to brokers.

**Comment by bgabela** [comment]
Yeah I marked it as "wontfix" earlier, but those who need it are welcome to fork an mt4 version
